### DISCUSSION TITLE
compare with gonum???

### DISCUSSION THREAD

**Discussion by rlacjfjin** [cpmech/gosl#54] [Q&A] [answered]
Hi,
I want to use matrix part of the gosl.  In my code, I suppose using LU factorization to solving Ax=b or trans(A)x=b  such form.
Because of calling frequency of solving linear systems, I hope to have efficient package to use it. 
I find two main packages, GOSL and GONUM, both are support matrix computations, but I can't sure which is best for my cases. Should you give me some advices?
In fact, I want to write a linear programming solver using go language for exercise....
Thanks

**Comment by cpmech** [comment, marked as answer]
Hi there, I had a look on gonum but cannot really give you much information on your query. I suppose you'll have to try both... Kind regards. Dorival.
